apps/api_web/lib/api_web/rate_limiter/memcache.ex

```diff
@@ -5,15 +5,9 @@ defmodule ApiWeb.RateLimiter.Memcache do
   @behaviour ApiWeb.RateLimiter.Limiter
   alias ApiWeb.RateLimiter.Memcache.Supervisor
 
-  @impl ApiWeb.RateLimiter.Limiter
-  def start_link(opts) do
-    clear_interval_ms = Keyword.fetch!(opts, :clear_interval)
-    clear_interval = div(clear_interval_ms, 1000)
-
-    connection_opts =
-      [ttl: clear_interval * 2] ++ ApiWeb.config(RateLimiter.Memcache, :connection_opts)
-
-    Supervisor.start_link(connection_opts)
+  @impl true
+  def start_link(_) do
+    {:ok, self()}
   end
```

Comment:
I have removed `start_link/1` from the behavior as it is no longer necessary here. The code that ultimately invokes the `start_link` has been updated to check if the function exists first instead, to handle the ETS based implementation.

Reply:
> question: what's the intention of this change? It appears to turn this `start_link` into a no-op, instead of starting the process defined by the current module, as is convention.

I found that it was simplest to fix this by simply making the memcache implementation a GenServer, to get rid of this misleading code.